src/nxt_http_variables.c

```diff
@@ -27,6 +29,10 @@ static nxt_var_decl_t  nxt_http_vars[] = {
     { nxt_string("host"),
       &nxt_http_var_host,
       0 },
+
+    { nxt_string("$$"),
```

Comment:
Other variables are using here the variable name without the `$` prefix.  Do we need to use `$$` here for some reason, or can we use simply `$`?  Does it complicate the code in some other place (or maybe make it simpler)?  Does it have any implications?

Reply:
Actually I had it using just '$' originally, easy to adjust.